server/rosetta/converter.go

```diff
@@ -334,12 +335,21 @@ func sdkEventToBalanceOperations(status string, event abci.Event) (operations []
 		isSub             bool
 	)
 
+	decEvents := make([]string, len(event.Attributes))
```

Comment:
nit: `decAttrs` or simply `attrs` sounds like a better name. `decEvents` made me think this was an slice of events instead of an slice of event's attributes.

Reply:
done!